_shared/hooks/post_gen_project.py

```diff
@@ -20,7 +20,18 @@ def remove_conditional_files():
     {% endif %}
 
     {% if cookiecutter.get("postgres") != "yes" %}
-    paths_to_remove.extend(["bin/create-db", "bin/make_db", "conf/alembic.ini"])
+    paths_to_remove.extend([
+        "bin/create-db",
+        "bin/make_db",
+        "conf/alembic.ini",
+        "{{ cookiecutter.package_name }}/db.py",
+        "{{ cookiecutter.package_name }}/models/__init__.py",
+        "tests/factories/__init__.py",
+        "tests/factories/factoryboy_sqlalchemy_session.py",
+        "tests/pytest_plugins/db.py",
+        "tests/unit/pytest_plugins/db.py",
+        "tests/functional/pytest_plugins/db.py",
+    ])
```

Comment:
This PR adds several new files that're only needed when the `postgres` option is on. Adding them to this list (along with the already-existing `postgres`-only files) ensures that these files won't be added to projects that don't use the `postgres` option